ring.go

```diff
@@ -30,6 +31,24 @@ type RingOptions struct {
 	// Shard is considered down after 3 subsequent failed checks.
 	HeartbeatFrequency time.Duration
 
+	// Hash function used in consistent hash, will use crc32.ChecksumIEEE
+	// from hash/crc32 package if not specified
+	HashFn Hash
```

Comment:
Can you please rename this HashFn -> Hash? It will be consistent with type `consistenthash.Hash`